src/framework/theme/modal/modal.service.tsx

```diff
@@ -71,10 +71,9 @@ class ModalServiceType {
     this.panel = null;
   }
 
-  public show(element: React.ReactElement<ModalComponentCloseProps>,
-              closeOnBackDrop: boolean = false): string {
+  public show(config: ModalPresentingConfig): string {
```

Comment:
It would be better to have another syntax:
```
show(element, config)
```

Reply:
Done.